### PR TITLE
Changelog fixes

### PR DESCRIPTION
**Pull request by amyblais**
- There was a request to continue to duplicate important upgrade notes in the changelog https://community.mattermost.com/core/pl/pnx99rurw38bjxcm3bek968jia.
 - Added a known issue to the desktop app changelog.

### CODE REVIEW

File: source/deploy/mattermost-changelog.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
   - Original 9.2.0 release
 
 ### Important Upgrade Notes
+ - Fixed data retention policies to run jobs when any custom retention policy is enabled even when the global retention policy is set to “keep-forever”. Before this fix, the enabled custom data retention policies wouldn’t run as long as the global data retention policy was set to “keep-forever” or was disabled. After the fix, the custom data retention policies will run automatically even when the global data retention policy is set to “keep-forever”. Once the server is upgraded, posts may unintentionally be deleted. Admins should make sure to disable all custom data retention policies before upgrading, and then re-enable them again after upgrading.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:34]:
```suggestion
 - Fixed data retention policies to run jobs when any custom retention policy is enabled even when the global retention policy is set to **keep-forever**. Before this fix, the enabled custom data retention policies wouldn’t run as long as the global data retention policy was set to **keep-forever** or was disabled. After the fix, the custom data retention policies will run automatically even when the global data retention policy is set to **keep-forever**. Once the server is upgraded, posts may unintentionally be deleted. Admins should make sure to disable all custom data retention policies before upgrading, and then re-enable them again after upgrading.
```

**Review by cwarnermm — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 2]
```diff
@@ -31,6 +31,7 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
   - Original 9.2.0 release
 
 ### Important Upgrade Notes
+ - Fixed data retention policies to run jobs when any custom retention policy is enabled even when the global retention policy is set to **keep-forever**. Before this fix, the enabled custom data retention policies wouldn’t run as long as the global data retention policy was set to **keep-forever** or was disabled. After the fix, the custom data retention policies will run automatically even when the global data retention policy is set to **keep-forever**. Once the server is upgraded, posts may unintentionally be deleted. Admins should make sure to disable all custom data retention policies before upgrading, and then re-enable them again after upgrading.
  - Please read the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) before upgrading.
```
Comment (hanzei) [inline source/deploy/mattermost-changelog.md:35]: A bit of a late review: Given that the important upgrade notes for this release are already here, this sentence needs to be more accurate as it suggests that there are more notices for this release. Can we clarify that the link contains only additional notes for other releases?

Comment (amyblais) [reply source/deploy/mattermost-changelog.md:35]: I'll make this edit.

Comment (hanzei) [reply source/deploy/mattermost-changelog.md:35]: Thanks @amyblais

**Pull request merged**